lib/vets/attributes.rb

```diff
@@ -61,11 +61,14 @@ def define_getter(name, default)
           return nil unless defined?(default)
 
           # if there's a default, assign the default value
-          if default.is_a?(Symbol) && respond_to?(default)
-            send(default)
-          else
-            default
-          end
+          value = if default.is_a?(Symbol) && respond_to?(default)
+                    send(default)
+                  else
+                    default.dup
+                  end
+
+          instance_variable_set("@#{name}", value)
```

Comment:
saw this in the other PR I approved, and didnt say anything, but this setting as an instance variable is required? It wasnt there before? Is it to do with it being accessible as a method vs as a hash now?